includes/import-products.php

```diff
@@ -114,6 +138,16 @@ function dw_add_woocommerce_product($course) {
 	wp_set_object_terms($product->get_id(), $startDate->format('H:i'), 'pa_tijd', false);
 	wp_set_object_terms($product->get_id(), $course['parts'][0]['slots'][0]['city'], 'pa_locatie', false);
 
+	foreach($course['parts'] as $part) {
+		$startDate = new DateTime($part['slots'][0]['startDate']);
+		wp_set_object_terms(
+			$product->get_id(),
+			date_i18n('d F Y', $startDate->getTimestamp()) . ' ' . $startDate->format('H:i'),
+			'pa_slot_time',
```

Comment:
I don't really understand what the `pa_slot_time` stuff does?

Reply:
It creates custom tags for WooCommerce product.

Reply:
It creates custom tags for WooCommerce product(See picture in slack).